deployment.yaml.erb

```diff
@@ -34,22 +37,25 @@ data:
     <%= line -%>
 <% end -%>
 <% end -%>
+
 <%- @nodes.each do |node| -%>
 <%= set_node_template_vars(node.values.first) -%>
+<%- (0..@replicate).each do |nodeNumber| -%>
```

Comment:
maybe it's worth to set a new variable under each block, sth like  `node_name = @Node_UserIdent + nodeNumber.to_s` and use that one instead of repeating the addition and `to_s` cast over and over again

Reply:
Good idea, not sure why I didn't do that in the first place